<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LicenseDialogUI</class>
 <widget class="QDialog" name="LicenseDialogUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>518</width>
    <height>434</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Licenses</string>
  </property>
  <layout class="QVBoxLayout">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="margin">
    <number>6</number>
   </property>
   <item>
    <widget class="RecordTreeView" name="mTreeView"/>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="mNewMethodButton">
       <property name="text">
        <string>New</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mRemoveMethodButton">
       <property name="text">
        <string>Remove</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>121</width>
         <height>14</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="mOKButton">
       <property name="text">
        <string>&amp;OK</string>
       </property>
       <property name="shortcut">
        <string>Alt+O</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mCancelButton">
       <property name="text">
        <string>&amp;Cancel</string>
       </property>
       <property name="shortcut">
        <string>Alt+C</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>RecordTreeView</class>
   <extends>QTreeView</extends>
   <header>recordtreeview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>mCancelButton</sender>
   <signal>clicked()</signal>
   <receiver>LicenseDialogUI</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>473</x>
     <y>415</y>
    </hint>
    <hint type="destinationlabel">
     <x>258</x>
     <y>216</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mOKButton</sender>
   <signal>clicked()</signal>
   <receiver>LicenseDialogUI</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>392</x>
     <y>415</y>
    </hint>
    <hint type="destinationlabel">
     <x>258</x>
     <y>216</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
